modules/ROOT/pages/sysconfig-network-configuration.adoc

```diff
@@ -244,7 +275,7 @@ ip=${ip}::${gateway}:${netmask}:${hostname}:${interface}:none:${nameserver}
 ip=10.10.10.10::10.10.10.1:255.255.255.0:myhostname:ens2:none:8.8.8.8
 ----
 
-==== Butane config
+==== Butane NetworkManager config
```

Comment:
To make it a little more clear:

```suggestion
==== Butane NetworkManager keyfile config
```

If we make this change then apply everywhere else too